### PR TITLE
Update nicklist when team member presence changes

### PR DESCRIPTION
**Pull request by marceldev89**
Depends on #399

### CODE REVIEW

File: wee_slack.py [hunk 1]
```diff
@@ -2151,7 +2151,15 @@ def process_manual_presence_change(message_json, eventrouter, **kwargs):
...
-    kwargs["user"].presence = message_json["presence"]
+    if "user" in kwargs:
+        user = kwargs["user"]
+        team = kwargs["team"]
+        user.presence = message_json["presence"]
+
+        for c in team.channels:
+            c = team.channels[c]
+            if user.id in c.members:
+                c.update_nicklist(user.id)
```
Comment (tfheen) [inline wee_slack.py]: This should probably be a function on the team, just "update presence for this user" and the team is responsible for iterating over any channels the user is in.

Comment (marceldev89) [inline wee_slack.py]: Will do. :)

Comment (marceldev89) [inline wee_slack.py]: Might as well just move everything to SlackTeam, right?

**Comment by tfheen** [comment]
Thanks for the patch, tested and merged. :+1:

**Pull request merged**